cfme/intelligence/reports/dashboards.py

```diff
@@ -243,18 +245,16 @@ def update(self, updates):
             updates: Provided by update() context manager.
         """
         view = navigate_to(self, "Edit")
+        if "widgets" in updates:
+            updates["widget_picker"] = updates.pop("widgets")
         changed = view.fill(updates)
         if changed:
             view.save_button.click()
         else:
             view.cancel_button.click()
         view = self.create_view(DefaultDashboardDetailsView)
         assert view.is_displayed
-        if changed:
-            view.flash.assert_success_message('Dashboard "{}" was saved'.format(self.name))
```

Comment:
**Required**: Better to move it to one of the tests, or add  a TODO to be moved to tests in scope with another flash messages